src/blocks/YoutubeVideoBlock/YoutubeVideoBlock.tsx

```diff
@@ -122,11 +127,13 @@ const VideoFrame = styled.iframe<{ width?: number; height?: number }>`
 interface YoutubeVideoBlockProps extends BrandPivotBaseBlockProps {
   overlay_image: Image
   video_id: string
+  caption?: MarkdownHtmlComponent
```

Comment:
should we add this to the `storyblok/components.json`  file?

Reply:
Yes! I didn't know what the best way was to get it locally but I will do a sync from staging as you suggested :)

Reply:
The other way around haha, fetch to my local setup. Updated now anyway :)